CI.md

```diff
@@ -125,6 +125,24 @@ Service Account shall have following permissions ([IAM roles](https://cloud.goog
 | Java Wordcount Direct Runner | Runs Java WordCount example with Direct Runner.                                               | Yes              | Yes                   | Yes           | -                        |
 | Java Wordcount Dataflow      | Runs Java WordCount example with DataFlow Runner.                                             | -                | Yes                   | Yes           | Yes                      |
 
+### PostCommit Workflows
+
+#### XVR Spark - [job-postcommit-xvr-spark.yml](.github/workflows/job-postcommit-xvr-spark.yml)
```

Comment:
Rather than having a bunch of distinct sections can we just make one larger table? I don't think each of these sections add much

(added same comment in other PRs)

Reply:
Yes, changed it. Thanks!